src/Aqovia.Utilities.SagaMachine/StatePersistance/RedisKeyValueStore.cs

```diff
@@ -29,15 +31,21 @@ public RedisKeyValueStore()
                 configurationOptions.EndPoints.Add(redisHost.Host, redisHost.CachePort);
             }
             _redis = ConnectionMultiplexer.Connect(configurationOptions);
-            
+
         }
 
         private IDatabase GetDatabase()
-        {            
-            IDatabase db = _redis.GetDatabase(_redisConfiguration.Database);            
+        {
+            IDatabase db = _redis.GetDatabase(_redisConfiguration.Database);
             return db;
         }
 
+        public IServer GetServer()
```

Comment:
New member but not used anywhere in this pull request?
I guess a leftover from prototyping that can now be removed?